ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/accordion/.content.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@rakeshkumar-verma : so u have checked in all component references too . Could have been avoided but ok for now.

Reply:
Will remove once we have project specific code